app/istqb-ct-mat-complete-guide/istqb-ct-mat-complete-guide.css

```
@@ -0,0 +1,991 @@
+                font-size: 0.78rem;
+            }
+            .meta-pill .val {
+                color: var(--neon-green);
+                font-weight: 700;
+                font-size: 1rem;
+            }
+            .meta-pill .lbl {
+                color: var(--text-muted);
+                font-size: 0.68rem;
+            }
+            @keyframes fadeInDown {
```

Stylelint (17.11.0)

[error] Expected keyframe name "fadeInDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)